tests/integration/ipsec_test.py

```diff
@@ -590,6 +590,38 @@ def test_ipsec_ipv4_libreswan_cert_auth_add_and_remove(
     )
 
 
+def test_ipsec_ipv4_libreswan_remote_cert_auth_add_and_remove(
```

Comment:
We haven't decided to use `remote` to replace `right` yet.

So please use `test_ipsec_ipv4_libreswan_rightcert()`.

Reply:
Meanwhile, please mark this test case as xfail as the required NetworkManager-libreswan is not shipped to our test environment yet.